pkg/conversion/mapi2capi/util.go

```diff
@@ -22,10 +22,16 @@ import (
 	mapiv1beta1 "github.com/openshift/api/machine/v1beta1"
 	"github.com/openshift/cluster-capi-operator/pkg/util"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/util/sets"
 	"k8s.io/apimachinery/pkg/util/validation/field"
 	clusterv1 "sigs.k8s.io/cluster-api/api/v1beta1"
 )
 
+var (
+	skippedMAPILabels      = sets.New("machine.openshift.io/instance-type", "machine.openshift.io/region", "machine.openshift.io/zone")
+	skippedMAPIAnnotations = sets.New("machine.openshift.io/instance-state")
```

Comment:
Hm, could we dynamically pull these from a constant that we can use both here and in the implementation?

Reply:
using the consts now. Also adjusted the names.